### ISSUE TITLE
Time.current.strftime should not be an offense

### ISSUE THREAD

**Issue by filipegiusti** [bbatsov/rubocop#2026]
On code
```ruby
timestamp = Time.current.strftime('%Y%m%d_%H%M%S')
File.open("report-#{timestamp}", 'w')
```
at rubocop version
```
$ rubocop -v
0.32.1
```
I'm getting
```
Do not use Time.strftime without zone. Use one of Time.strftime.in_time_zone, Time.strftime.utc, Time.strftime.iso8601, Time.strftime.jisx0301, Time.strftime.rfc3339, Time.strftime.to_i, Time.strftime.to_f instead.
```
But I don't want to have a Time object (as returned by `in_time_zone`), I only want my formatted string and I'm using current to set the right timezone.

**Comment by mikegee** [comment]
I confirmed the reported behavior in the latest master just now.

I like to use `Time.current`, too. It is timezone aware and should be supported by Rubocop, but isn't yet.  http://api.rubyonrails.org/classes/Time.html#method-c-current

**Comment by ClairePitman** [comment]
The docs say that if Time.zone or config.time_zone are not set it just returns Time.now.
I think that this would be ok when EnforcedStyle is acceptable but otherwise it seems like a step back toward Time.now.

**Comment by bbatsov** [comment]
//cc @palkan 

I've never received more issue reports than about this cop. Seems there's no good way to check for the time zone problems reliably, so I guess we should just disable it by default.

**Comment by palkan** [comment]
@bbatsov Maybe you're right. Too much edge cases.

But nevertheless I'll add support for `Time.current` as acceptable.

**Comment by bbatsov** [comment]
Yep, you should.
